src/app/blog/index.html

```diff
@@ -3,7 +3,34 @@
   pageTitle="SKY UX Blog"
   windowTitle="SKY UX Blog">
   
-<article>
+	<article class="blog-post">
+		<header>
+			<h2 class="sky-section-heading">Design update — March 2018</h2>
+			<p>SKY UX releases 2.10.0-2.11.0</p>
+		</header>
+		<section>
+			<h3 class="sky-subsection-heading">Recently completed</h3>
+				<h4 class="sky-emphasized"><a routerLink="/components/lookup">Lookup</a></h4>
+					<p>The lookup component combines typeahead search functionality with the option to select one or multiple items for data entry.</p>
+				<h4 class="sky-emphasized"><a routerLink="/components/flyout">Flyout</a></h4>
+					<p>Added the ability to resize the flyout panel.</p>
```

Comment:
Based on the changelog for 2.11, I think it's just the width, so I'd add "the width of" after "resize."